src/js/Calculator.js

```diff
@@ -0,0 +1,105 @@
+import { OPERATIONS, MESSAGES, MAX_NUMBER_LENGTH } from "./constants.js";
+import ComputedData from "./ComputedData.js";
+import Display from "./Display.js";
+
+export default function Calculator() {
+  const state = {
+    clickedDigits: [0, 0],
+    operation: "",
+    result: "",
+  };
+
+  const render = () => {
+    onClickDigit();
+    onClickOperation();
+    onClickAC();
+  };
+
+  const $ = selector => document.querySelector(selector);
+  const $digits = $(".digits");
+  const $operations = $(".operations");
+  const $modifier = $(".modifier");
+
+  const onClickDigit = () => {
+    $digits.addEventListener("click", e => {
```

Comment:
해당 기능에서 이벤트(`e`) 부분에서 필요한 속성이 target 인 거 같은데, 그러면 target 만 받아서 처리하는 걸 추천합니다.
왜냐하면 이벤트 객체는 많은 속성들을 가지고 있어, 이벤트 발생할 때마다 불 필요한 정보까지 전달하면 그만큼 리소스가 손해보는 걸로 알고 있습니다. 
아래와 같이 수정할 수 있을 거 같아요 
```suggestion
    $digits.addEventListener("click", { target } => {
      addDigit(target.value);
    });
  };
```

Reply:
아하! 이렇게 사용하는 이유를 몰랐는데, 리소스를 줄이기 위해 이런 방법으로 사용하는 거였군요. 
중요한 지점 알려주셔서 감사합니다!